examples/habana/streaming_llm/eval_bf16_streaming.sh

```diff
@@ -0,0 +1,33 @@
+MODEL_NAME_OR_PATH=${MODEL:-meta-llama/Llama-2-13b-hf}
```

Comment:
what is the difference between this and ```ru_yi_34b_xxx```? can you merge them into one script?

Reply:
I can rename `run_yi_34b_xxx` to "run_xx_streaming" like eval. And we can set the default model to `YI-34B-Chat`. Is that ok for you?

Reply:
> I can rename `run_yi_34b_xxx` to "run_xx_streaming" like eval. And we can set the default model to `YI-34B-Chat`. Is that ok for you?

ok for me

Reply:
updated in https://github.com/intel/intel-extension-for-transformers/pull/1558/commits/15a038a70f9cac59cbefed16c303c8c43eee240a